src/odin/maneuver.cc

```diff
@@ -764,6 +766,36 @@ void Maneuver::set_contains_obvious_maneuver(bool contains_obvious_maneuver) {
   contains_obvious_maneuver_ = contains_obvious_maneuver;
 }
 
+bool Maneuver::has_combined_enter_exit_roundabout() const {
```

Comment:
Should we output this property in `Maneuver::ToString` as well?

Reply:
we could - currently, the value would not be populated until after to trace output

Reply:
> we could - currently, the value would not be populated until after to trace output

Okay, I pointed out because I've seen you usually add new attributes to the `ToString` function. I'll leave it to you. The changes look good.